### PR TITLE
Small fixes and tweaks 

### PR DESCRIPTION
**Pull request by luzpaz**
+ Small whitespace fixes 
+ Cleaned some redundant Python imports
+ found a hardcoded path to .dat file that needs to be modified

### CODE REVIEW

File: airPlanePanelObject.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def __init__(self, obj):
         obj.addProperty("App::PropertyLength","Length","WingPanel","Length of the wing").Length=1.0
         obj.addProperty("App::PropertyLength","Width","WingPanel","Width of the wing").Width=1.0
         obj.addProperty("App::PropertyLength","Height","WingPanel", "Height of the wing").Height=1.0
-        obj.addProperty("App::PropertyStringList","PanelProfil","WingPanel","Profil type").PanelProfil=[u"/Users/fredericnivoix/Library/Preferences/FreeCAD/Mod/AirPlaneDesign/wingribprofil/e207.dat"]
+        obj.addProperty("App::PropertyStringList","PanelProfil","WingPanel","Profil type").PanelProfil=[u"Mod/AirPlaneDesign/wingribprofil/e207.dat"]
```
Comment (luzpaz) [inline airPlanePanelObject.py:166]: please review

File: airPlaneWingUI.py [hunk 2]
```diff
@@ -40,21 +40,26 @@ def loadPanelTable(self):
         _wingRibProfilDir=FreeCAD.getUserAppDataDir()+ 'Mod/AirPlaneDesign/wingribprofil'
         #self.form.NumberOfPanel.setItem=5
         # Numéro du panneau, profil, fichier profil, , corde emplature, corde saumon, longueur paneau, X emplature, X saumon, Y emplature, Y saumon, Z emplature, Z saumon, X Rotation, Y Rotation, Z Rotation
-        initPanelTable =[
-                         ["1","Eppler207",_wingRibProfilDir+u"/naca/naca2412.dat","250","222","122","0","0","0.0","-54.","0","0","22"],["2","Eppler207",_wingRibProfilDir+u"/naca/naca2412.dat","222","196","35.","0","0","-54","-54","0","54","0"],["3","Eppler205",_wingRibProfilDir+u"/naca/naca2412.dat","196","146","456","0","0","-54","12","0","0","81"],["4","Eppler205",_wingRibProfilDir+u"/naca/naca2412.dat","146","100","10","0","0.","12","12","12","0","0"],["5","Eppler205",_wingRibProfilDir+u"/naca/naca2412.dat","100","100","10","0","0.","12","12","0","0","0"]
+        # Panel #, profile, profile file....X Rot(ation), Y Rot, Z Rot
```
Comment (luzpaz) [inline airPlaneWingUI.py:43]: Do you mind translating this in to english?

**Comment by luzpaz** [comment]
I was hoping that you would answer before merging so I could fix it. I don't think all the changes I made were correct. That's why I pointed them out

**Comment by FredsFactory** [comment]
don't worry  it was a wrong manipulation with my phone ... I will check  tonight...

File: airPlaneWingUI.py (hunk 2 above)
Comment (FredsFactory) [reply airPlaneWingUI.py:43]: Yes of course.....done...

**Comment by FredsFactory** [comment]
Thanks for your contribution.

**Pull request merged**